e2e/specs/st_hello.spec.js

```diff
@@ -72,7 +72,7 @@ describe("hello", () => {
             );
 
             // Wait for the animation to end.
-            cy.get(".stText").contains("100% Complete");
+            cy.get([(data - testid = "stText")]).contains("100% Complete");
```

Comment:
nit: style (here and below), would prefer the consistent `"[data-testid='stText']"`